nf_core/components/update.py

```diff
@@ -362,7 +362,8 @@ def get_single_component_info(self, component):
             raise LookupError(f"No {self.component_type} installed from '{repo_url}'")
 
         choices = [
-            component if directory == "nf-core" else f"{directory}/{component}" for directory, component in components
+            component if directory == self.modules_repo.repo_path else f"{directory}/{component}"
+            for directory, component in components
```

Comment:
```suggestion
            component for _, component in components
```
I'm not sure how to test that right now, but I think we can directly remove the whole if/else. Actually, this was a comment on the original PR, but we decided to change it later. 
This will have some problems if there are two modules with the same name, but TBH I think we will have many other problems if that happens, so we should handle it differently. 
If a user provides the name of a module without specifying the directory, the first code will fail. Not sure if that was the case for this particular issue, but I would say everyone will expect it to work with only the tool name.

Reply:
module or subworkflow name! 😄

Reply:
I don't think this makes a difference unless we change how we initially get `components` right? (Which right now is passed a `repo_url`).

This is part of a wider change I would like to make (probably in 2.8) where other than `install` we shouldn't need to rely on the remote repo being passed to the command with `--git-repo` (we already made it that way for lint I think?)

I have some experimental work that returns displays the module/subwf choice with it's "org" too, but for now that still didn't fix underlying issues if two modules/subwf had the same name so I haven't progressed it

Reply:
Makes sense, let's leave these big changes for 2.8 👍